components/mixins/place.js

```diff
@@ -14,11 +14,9 @@ export default {
     },
 
     contact() {
+      var PhoneNumber = require('awesome-phonenumber');
```

Comment:
You should move it to the top of the `<script>` tag and use the modern import syntax.

Reply:
You mean moving the import to the top of the file, or moving it to another file?